File: HNQQPopMenu/Source/HNPopMenuTableViewCell.m
//
//  HNPopMenuTableViewCell.m
//  HNPopMenu
//
//  Created by ZakariyyaSv on 16/4/2.
//  Copyright © 2016年 ZakariyyaSv. All rights reserved.
//

#import "HNPopMenuTableViewCell.h"

@implementation HNPopMenuTableViewCell

- (void)awakeFromNib {
    [super awakeFromNib];
    // Initialization code
}

- (instancetype)initWithTableView:(UITableView *)tableView{
    static NSString *identifier = @"popmenuTableViewCell";
    HNPopMenuTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:identifier];
    if (!cell) {
        cell = [[HNPopMenuTableViewCell alloc] initWithStyle:UITableViewCellStyleDefault reuseIdentifier:identifier];
        cell.textLabel.font = [UIFont systemFontOfSize:15.0f];
    }
    return cell;
}

@end

File: HNQQPopMenu-swift/HNQQPopMenu-swift/HNQQPopMenu/HNPopMenuTableViewCell.swift
//
//  HNPopMenuTableViewCell.swift
//  HNQQPopMenu-swift
//
//  Created by zakariyyaSv on 16/4/22.
//  Copyright © 2016年 Zakariyya. All rights reserved.
//

import UIKit

class HNPopMenuTableViewCell: UITableViewCell {
    
    class func initWithTableView(tableView:UITableView) -> HNPopMenuTableViewCell{
        let identifier = "popMenuTableViewCell"
        var cell = tableView.dequeueReusableCellWithIdentifier(identifier) as? HNPopMenuTableViewCell
        if (cell == nil) {
            cell = HNPopMenuTableViewCell.init(style: UITableViewCellStyle.Default, reuseIdentifier: identifier)
            cell?.textLabel?.font = UIFont.systemFontOfSize(15.0)
        }
        return cell!
    }

    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }

}
